cli

```diff
@@ -48,6 +48,9 @@ const options = argv.usage('Usage: $0 path [options]')
     .describe('http-always-get', 'Use always HTTP GET requests, by default HEAD is used for pages without any anchors')
     .boolean('warn-name-attr')
     .describe('warn-name-attr', 'show warning if name attribute instead of id was used for an anchor')
+    .describe('external-link-cache', "Directory to store the non failing responses. If none is specified responses won't be cached.")
```

Comment:
Feel free to come up with some better names for the options, but other than that I'm happy :smile: